exo-go/src/types/docker_config.go

```diff
@@ -4,7 +4,7 @@ package types
 // docker-compose.yml
 type DockerConfig struct {
 	Image         string            `yaml:",omitempty"`
-	Build         string            `yaml:",omitempty"`
+	Build         map[string]string `yaml:",omitempty"`
```

Comment:
I'm not sure if I understood you correctly. Did you mean setting `build` to `../..` (since it needs to be relative to `docker-compose.yml`)? I don't think that works since `docker-compose` will try to look for a `service/Dockerfile`. If we set `build` to `../tests`, it will find the `Dockerfile` but it doesn't have access to `package.json` which is in `../..`.  I believe the user will also have to change the paths in `Dockerfile` (e.g. `npm install ../package.json`).